cmds/check-it.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var request = require('request');
+var logger = require('winston-color');
+var npm = require('npm');
+var shell = require("shelljs");
+
+var fix = false;
+var scrub = false;
+var location, failure, manifest;
+
+module.exports = function(program) {
+
+  program
+    .command('*')
+    .description('Check that your project repo meets CFPB\'s open source checklist requirements.')
+    .action(function(loc) {
+      location = loc || '.';
+      fix = program.fix;
+      scrub = program.scrub;
+      [checkPII, checkTERMS, checkCONTRIBUTING, checkCHANGELOG, scrubGHEReferences].forEach(function(func) {
+        func(function(err, msg) {
+          if (err) {
+            return logger.error(err);
+            process.exit(1);
+          }
+          return logger.info(msg);
+        });
+      });
+    });
+
+};
+
+function checkPII(cb) {
+  // Has PII been removed?
+}
+
+function checkTERMS(cb) {
+  fs.readFile(path.join(location, './TERMS.md'), 'utf8', function (err, data) {
+    if (err) return handleIssue(1, cb);
+    cb(null, 'Your TERMS.md file looks good.');
+  });
+}
+
+function checkCONTRIBUTING(cb) {
+  fs.readFile(path.join(location, './CONTRIBUTING.md'), 'utf8', function (err, data) {
+    if (err) return handleIssue(2, cb);
+    cb(null, 'Your CONTRIBUTING.md file seems to be in order.');
+  });
+}
+
+function checkCHANGELOG(cb) {
+  fs.readFile(path.join(location, './CHANGELOG.md'), 'utf8', function (err, data) {
+    if (err) return handleIssue(3, cb);
+    cb(null, 'Your CHANGELOG.md file is good to go.');
+  });
+}
+
+function scrubGHEReferences(cb) {
+  if (scrub) {
+    fs.readFile(path.join(location, './.env'), 'utf8', function (err, data) {
+      if (err) return handleIssue(4, cb);
+      var envPath = path.join(location, './.env');
+      shell.exec('. ' + envPath);
```

Comment:
to set the GHE_URL variable and run the scrub git history searches w/it. let me know if you know a better way